specification/storage/resource-manager/Microsoft.Storage/stable/2025-08-01/blob.json

```diff
@@ -0,0 +1,1579 @@
+{
+  "swagger": "2.0",
```

Comment:
To avoid customer issue, before merge this PR, please make sure:

1. The new API version is available in all regions (include usgov, mooncake ...)
2. All new API changes are available in all regions .
 - If some API change will not be available in some regions by design, please point it out, and share the error message when user use the API in not supported regions. The error message should be clear and actionable.